Kickstarter-iOS/Views/Controllers/ManagePledgePaymentMethodView.swift

```diff
@@ -3,12 +3,22 @@ import Library
 import Prelude
 import UIKit
 
+protocol ManagePledgePaymentMethodViewDelegate: AnyObject {
+  func managePledgePaymentMethodViewDidTapFixButton(_ view: ManagePledgePaymentMethodView)
+}
+
 final class ManagePledgePaymentMethodView: UIView {
+  weak var delegate: ManagePledgePaymentMethodViewDelegate?
+
   // MARK: - Properties
 
   private lazy var cardLabelsStackView: UIStackView = { UIStackView(frame: .zero) }()
   private lazy var cardNumberLabel: UILabel = { UILabel(frame: .zero) }()
   private lazy var expirationDateLabel: UILabel = { UILabel(frame: .zero) }()
+  private lazy var fixButton: UIButton = { UIButton(type: .custom)
+    |> \.translatesAutoresizingMaskIntoConstraints .~ false
```

Comment:
Don't think we need this when we're adding directly to a stackview.

Reply:
We need it cuz we set a height and width in `setupConstraints`

Reply:
Oh right but it seems that simply by adding the view to a stackview it sets `translatesAutoresizingMaskIntoConstraints` to `false`. Anyway, not a big deal but if you did feel like removing it everything will still work.